### PR TITLE
Added badge to menu button

### PR DESCRIPTION
**Pull request by ErlingMK**
## Added / Fixed

| Properties | Explanation | Type |
| --- | --- | --- |
| AttachBadge | Toggles if badge should show or not | bool |
| BadgeColor | Sets color of badge | Color |
| BadgeCounter | Changes content of badge | int |

## Todo List

- [x] I have tested on an Android device.
- [x] I have tested on an iOS device.
- [ ] I have added unit tests

>*Please add pictures / Gifs if possible*
>*Todo List can be checked by putting a `X` inside the brackets*
>*Remember to update our `wiki` after this PR is merged*

### CODE REVIEW

File: src/Samples/DIPS.Xamarin.UI.Samples/Controls/FloatingActionMenu/FloatingActionMenuPage.xaml [hunk 1]
```diff
@@ -26,21 +26,23 @@
                                               CloseOnOverlayTapped="{Binding Source={x:Reference shouldCloseOnTappedOverlayCheckbox}, Path=IsChecked}"
                                               IsVisible="{Binding Source={x:Reference toggleVisibilityCheckbox}, Path=IsChecked}">
                 <dxui:MenuButton BackgroundColor="LightSeaGreen"
-                                 Text="A"
+                                 Text="HEI"
```
Comment (haavamoa) [inline src/Samples/DIPS.Xamarin.UI.Samples/Controls/FloatingActionMenu/FloatingActionMenuPage.xaml]: Update samples with values that makes more sense :P Maybe the buttons text could have something like "Home", "About", "Notifications" etc.

**Comment by Illedan** [comment]
Gifs?

**Comment by ErlingMK** [comment]
Badge

File: src/DIPS.Xamarin.UI/Controls/FloatingActionMenu/MenuButton.xaml [hunk 2]
```diff
@@ -43,17 +45,42 @@
...
+                                FontSize="{Binding FontSize}"
+                                Text="{Binding Text}"
+                                TextColor="{Binding TextColor}"
+                                FontFamily="{Binding FontFamily}"
+                                Clicked="MenuButton_OnClicked" />
+
+        <Frame x:Name="BadgeFrame"
+               x:FieldModifier="internal"
+               Grid.Column="1"
+               IsVisible="{Binding IsBadgeVisible}"
```
Comment (haavamoa) [inline src/DIPS.Xamarin.UI/Controls/FloatingActionMenu/MenuButton.xaml]: Badge should not be visible if the behaviors "IsVisible" is set to false

**Pull request merged**